src/platforms/ruby/common/configuration/options.mdx

```diff
@@ -130,14 +130,13 @@ config.inspect_exception_causes_for_exclusion = true
 
 `logger`
 
-: The logger used by Sentry. Default is `Rails.logger` in Rails, otherwise an instance of `Sentry::Logger`.
+: The logger used by Sentry. Default is `Rails.logger` in Rails, otherwise an instance of `Sentry::Logger`. Make sure to change the logger level if you need debug output.
```

Comment:
```suggestion
: The logger used by Sentry. Default is `Rails.logger` in Rails, otherwise an instance of `Sentry::Logger`. Change the logger level if you need debug output.
```
The original is also fine by me, but I slightly prefer this more concise version